docs/pages/trigger_connectors.md

```diff
@@ -117,7 +117,7 @@ If you'd like more fine grained control, declare it as a function returning a pr
 module.exports = function (params, http) {
   return when.promise(function (resolve, reject) {
 
-    if (http.method === 'post') {
+    if (http.method === 'POST') {
```

Comment:
why not `if (http.method.toLowerCase() === 'post')`

Reply:
I want to make clear the verb is received in upper case, `.toLowerCase()` seem to imply it can be received in either case

Reply:
Also, in the HTTP specification, HTTP verbs are case sensitive. In _theory_ it would be possible to have different behaviours for `post` and `POST` (and `Post` and `PoSt`, etc.)

Reply:
(Danny I did the telnet test we discussed, the aws LB shuts down the connection if it sees a weird looking verb)